en/payloads/README.md

```diff
@@ -0,0 +1,109 @@
+# Payloads and Cameras
+
+PX4 supports a wide range of payloads and cameras.
+
+## Mapping Drones
+
+* [Camera triggering](../peripherals/camera.md) via GPIO out
+* [Camera triggering](../peripherals/camera.md) via PWM out
+* [Camera triggering](../peripherals/camera.md) via MAVLink out
+* [Camera timing](../peripherals/camera.md#camera_capture) feedback via hotshoe input
+
+## Cargo drones and alike: Servos / Outputs
+
+* Servo or GPIO triggering (via RC or via commands)
+
+### Example Mission (in QGC)
+
+Use MAV_CMD_DO_SET_ACTUATOR to trigger one of the payload actuators.
+
+### Example script (MAVSDK)
+
+This script sends a command to set the actuator and trigger the payload release on a servo:
+
+```
+#include <mavsdk/mavsdk.h>
+#include <mavsdk/plugins/mavlink_passthrough/mavlink_passthrough.h>
+#include <mavsdk/plugins/info/info.h>
+#include <chrono>
+#include <cstdint>
+#include <iostream>
+#include <future>
+#include <memory>
+
+using namespace mavsdk;
+
+void send_actuator(MavlinkPassthrough& mavlink_passthrough,
+        float value1, float value2, float value3);
+
+int main(int argc, char **argv)
+{
+    Mavsdk mavsdk;
+    std::string connection_url;
+    ConnectionResult connection_result;
+    float value1, value2, value3;
+
+    if (argc == 5) {
+        connection_url = argv[1];
+        connection_result = mavsdk.add_any_connection(connection_url);
+        value1 = std::stof(argv[2]);
```

Comment:
@julianoes These should set value to `NaN` if no value is passed in the argument - otherwise you're explicitly specifying a value. If not, what's the best way to do that?